src/locales/en.json

```diff
@@ -147,6 +147,10 @@
     "repeated": "Household member [{{ percentage }} chance of transmission per week]",
     "partner": "Partner / spouse [{{ percentage }} chance of transmission per week]",
     "indoor": "Indoor",
+    "filtered": "Indoors with a HEPA filter (flow rate 5x room size)",
```

Comment:
I wonder how we can help people make sense of this. Maybe here by adding an info alert below the input box if they select this option. Or we could add it to the whitepaper and link them to it. 

In either case, I'd love to be able to give them some basic instructions. Even something like "Google the flow rate of your HEPA filter. Make sure it's rated XYZ or above. Multiply the with x length x height of your room. If the flow rate is greater than 5x that room size per hour, then you're good to go." (Except I'm butchering the explanation, but you get the idea.)

Whatcha think about adding something like that.

Reply:
Yeah, I think a link to the whitepaper is better here.

Reply:
Done, ptal.